subgraphs/opensea/protocols/opensea-v2/src/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const PROTOCOL_SUBGRAPH_VERSION = "1.0.0";
+export const PROTOCOL_METHODOLOGY_VERSION = "1.0.0";
+export const PROTOCOL_MARKETPLACE_NAME = "OpenSea: Wyvern Exchange v2";
```

Comment:
I'd just call it `OpenSea v2`, which IMO is a more appropriate name for the marketplace